### DISCUSSION TITLE
Calibration mode: 100% to 80% does't work

### DISCUSSION THREAD

**Discussion by eighteentee** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#952] [Q&A]
The auto calibration mode of Al Dente is such a great idea! However, on my MacBook Pro M1 Pro Max, everything works as it should, that is, charge to 100%, discharge to 15%, charge to 100%... and then it stays there. It says it should hold for one hour, however, it just keeps on charging and doesn't discharge back to my target 80%. One of the giveaways is that the battery LED stays green. Why?

**Comment by masterzone3d** [comment]
Did you try to disable the Apple optimised battery charging feature?
<img width="604" alt="Screenshot 2023-05-18 at 17 47 35" src="https://github.com/davidwernhart/AlDente-Charge-Limiter/assets/132667740/2cf4ae11-5ae4-4e86-89fd-0a308e7242f9">

**Comment by MatthiasKerbl** [comment]
> The auto calibration mode of Al Dente is such a great idea! However, on my MacBook Pro M1 Pro Max, everything works as it should, that is, charge to 100%, discharge to 15%, charge to 100%... and then it stays there. It says it should hold for one hour, however, it just keeps on charging and doesn't discharge back to my target 80%. One of the giveaways is that the battery LED stays green. Why?

Hi @eighteentee,

I am currently not able to reproduce this bug.
How long did you wait for AlDente to discharge your MacBook again when 100% was reached? Sometimes it can take more than one hour.

Best,
Matthias

**Reply by eighteentee** [reply]
Ok. There is definitely no Apple power management on. I left it for 3+ hours after the initial reaching of 100% and it would not discharge. I had to unplug the power and let it 'manually' drop back to 80%, at which point I reconnected the power. Is there a way I can see internally what's happening when the 'hold' phase is active?
